bool limit_az
bool limit_el
bool emergency
float64 command_az
float64 command_el
float32 command_azspeed
float32 command_elspeed
string node_status
string from_node
float64 timestamp